gradle/libs.versions.toml

```diff
@@ -66,11 +66,14 @@ sqlDelight-coroutinesExt = { module = "com.squareup.sqldelight:coroutines-extens
 sqlDelight-native = { module = "com.squareup.sqldelight:native-driver", version.ref = "sqlDelight" }
 sqlDelight-runtime = { module = "com.squareup.sqldelight:runtime", version.ref = "sqlDelight" }
 
-touchlab-kermit = { module = "co.touchlab:kermit", version = "1.0.0" }
+touchlab-kermit = { module = "co.touchlab:kermit", version = "1.0.3" }
 touchlab-stately = { module = "co.touchlab:stately-common", version = "1.2.1" }
 
 turbine = { module = "app.cash.turbine:turbine", version = "0.7.0" }
 
+[plugins]
+gradleDependencyUpdate = "com.github.ben-manes.versions:0.41.0"
```

Comment:
This works alright, though I still wish there were better IDE support for gradle version catalogs instead of needing to manually run a task.

Do we have any sense of how much usage this is getting in the community? Just want to be careful about appearing to endorse it if we haven't vetted it some.

Reply:
Note sure what parameter do you use to get sense of popularity or usage in the community. Let me know if there is a specific way.

I initially found this used in Chris Banes' Tivi app
https://github.com/chrisbanes/tivi/blob/f0170cde2cffe6facb5762a8d2e5825d430d7378/gradle/libs.versions.toml#L27

and then noticed that the plugin itself is popular in general
![Screen Shot 2022-02-02 at 10 52 28 AM](https://user-images.githubusercontent.com/2466073/152189185-ccb9fed5-57db-45ed-9396-361b1b1c8746.png)